extra/src/main/java/aima/extra/probability/domain/AbstractFiniteDomain.java

```diff
@@ -0,0 +1,95 @@
+package aima.extra.probability.domain;
+
+import aima.extra.util.datastructure.BiMap;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Abstract base class of FiniteDomain implementations.
+ * 
+ * @author Nagaraj Poti
```

Comment:
As a rule of thumb if you are basing your code on pre-existing code you should keep the other author(s) in the list of authors. Order the authors based on who you believe contributed the most to the current design of the class - in this case you can list yourself first.

Note: I am guessing you just missed here as you have done this in the other code files.